### PR TITLE
Master fix

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

File: cypress/e2e/finance/funds/create-encumbrance-from-order.cy.js [hunk 1]
```diff
@@ -84,8 +84,10 @@ describe('ui-finance: Transactions', () => {
     FinanceHelp.selectFromResultsList();
     Funds.selectBudgetDetails();
     Funds.deleteBudgetViaActions();
+    cy.wait(1000);
```
Comment (TetianaParanich) [inline cypress/e2e/finance/funds/create-encumbrance-from-order.cy.js:87]: please add comment

File: cypress/e2e/finance/groups/add-multiple-funds-to-group.cy.js [hunk 2]
```diff
@@ -87,10 +87,14 @@ describe('ui-finance: Groups', () => {
     FinanceHelp.selectFromResultsList();
     Funds.selectBudgetDetails();
     Funds.deleteBudgetViaActions();
+    // Need to wait some time fo delating all items
+    cy.wait(1000);
     Funds.deleteFundViaApi(firstFund.id);
     Funds.deleteFundViaApi(secondFund.id);
+    cy.wait(1000);
```
Comment (TetianaParanich) [inline cypress/e2e/finance/groups/add-multiple-funds-to-group.cy.js:94]: please add comment

File: cypress/e2e/finance/groups/add-multiple-funds-to-group.cy.js [hunk 3]
```diff
@@ -87,10 +87,14 @@ describe('ui-finance: Groups', () => {
...
     Groups.deleteGroupViaApi(defaultGroup.id);
     Ledgers.deleteledgerViaApi(defaultLedger.id);
+    cy.wait(1000);
```
Comment (TetianaParanich) [inline cypress/e2e/finance/groups/add-multiple-funds-to-group.cy.js:97]: please add comment

File: cypress/support/fragments/receiving/receiving.js [hunk 4]
```diff
@@ -70,6 +70,7 @@ export default {
       SelectionOption('Main Library (KU/CC/DI/M) ').click(),
       receiveButton.click(),
     ]);
+    cy.wait(1000);
```
Comment (TetianaParanich) [inline cypress/support/fragments/receiving/receiving.js:73]: please add comment

File: cypress/support/fragments/receiving/receiving.js [hunk 5]
```diff
@@ -99,10 +101,12 @@ export default {
       Checkbox({ name: `${recievingFieldName}.checked` }).clickInput(),
       unreceiveButton.click(),
     ]);
+    cy.wait(1000);
```
Comment (TetianaParanich) [inline cypress/support/fragments/receiving/receiving.js:104]: please add comment

File: cypress/support/fragments/receiving/receiving.js [hunk 6]
```diff
@@ -99,10 +101,12 @@ export default {
...
     InteractorsTools.checkCalloutMessage(unreceivingSuccessful);
   },
 
   checkUnreceivedPiece: (rowNumber = 0, caption) => {
+    cy.wait(2000);
```
Comment (TetianaParanich) [inline cypress/support/fragments/receiving/receiving.js:109]: please add comment

**Review by TetianaParanich — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

**Pull request merged**